services.go

```diff
@@ -54,6 +63,47 @@ func applyService(c *cli.Context) error {
 	return nil
 }
 
+func updateServiceYamlContent(content string) string {
```

Comment:
@neelam-harness I saw a similar piece of code in your last PR i.e. https://github.com/harness/harness-cli/pull/32/files.

I think we should reuse this logic as its same in both services.go and infrastructure.go

Reply:
good catch! will move this to the utility class and reuse